apiRoutes/team_checklist.js

```diff
@@ -0,0 +1,118 @@
+const express = require("express")
+const router = express.Router()
+
+router.use(express.json())
+
+const { team_todolist } = require("../models")
+const { team_checklist } = require("../models")
+const { user } = require("../models")
+const { team } = require("../models")
```

Comment:
If didn't used pls delete it

Reply:
Yeah, it used once before I improve this api so you can delete it out now.